### PR TITLE
added geohash variable terms

### PR DESCRIPTION
**Pull request by zhengj2007**
Issue#: https://github.com/VEuPathDB-ontology/VEuPathDB-ontology/issues/122

### CODE REVIEW

**Comment by johnwjudkins** [comment]
Looks OK, except that the use of underscores in labels goes against [naming conventions](http://obofoundry.org/principles/fp-012-naming-conventions.html). I think we should use spaces instead.

**Comment by cstoeckert** [comment]
Propose that we use with spaces for the ontology label but include the label with the underscore for the alternative label. OK with you @bobular ?

File: src/ontology/eupath_dev.owl [hunk 1]
```diff
@@ -16092,6 +16104,513 @@ CD3T188G is a polymophism on the human CD36 gene.</obo:IAO_0000116>
...
+        <obo:IAO_0000233 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://github.com/VEuPathDB-ontology/VEuPathDB-ontology/issues/122</obo:IAO_0000233>
+        <obo:IAO_0000234 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">VectorBase</obo:IAO_0000234>
+        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">geohash_7</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/EUPATH_0043210 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/EUPATH_0043210">
```
Comment (bobular) [inline src/ontology/eupath_dev.owl:16235]: Hi Jie - sorry, actually we don't need the terms from 8 to 31.

**Review by bobular — CHANGES_REQUESTED** [review]
Looks good, I think.  Diffs of owl XML files aren't as nice as .obo files though.

(I had a quick look for ways to enhance git or github diffs for XML but didn't find anything.)

Just the one change requested (remove geohash_8 .. geohash_31 terms) thanks!

**Comment by bobular** [comment]
> Propose that we use with spaces for the ontology label but include the label with the underscore for the alternative label. OK with you @bobular ?

We can go with that. The primary label can be longer if you like!

From what I see in the web code, all variables are referred to by the EUPATH_0012345 style ID so the code will need a small lookup table from geohash level to EUPATH_nnnnnnn ID.  The code likely won't be able to use the 'geohash_N' alt labels.

**Comment by zhengj2007** [comment]
@johnwjudkins @bobular I made the changes according to your comments. Please check when you have time. Thanks!

**Comment by bobular** [comment]
Thanks Jie. I think we still need the geohash_32 term please.

File: src/ontology/eupath_dev.owl (hunk 1 above)
Comment (zhengj2007) [reply src/ontology/eupath_dev.owl:16235]: Removed them in the eupath_dev.owl

**Comment by zhengj2007** [comment]
@bobular I added 'geohash 32' back.

@johnwjudkins Could you please review the changes? Thanks!

**Review by johnwjudkins — APPROVED** [review]
OK

**Review by bobular — APPROVED** [review]

**Comment by bobular** [comment]
Ready to merge, thanks!

**Pull request merged**